### PR TITLE
Fixes to twifkak@ comments for , post submit.

### PR DESCRIPTION
**Pull request by banaag**
Fixes for twikfak@ comments in https://github.com/banaag/click-to-deploy/pull/5

### CODE REVIEW

File: k8s/amppackager/README.md [hunk 1]
```diff
@@ -32,10 +32,11 @@ Popular open stacks on Kubernetes packaged by Google.
 
 ![Architecture diagram](resources/amppackager-k8s-app-architecture.png)
 
-This application uses 2 instances of the AMP Packager that has a load balancer
-service in front that routes requests to the instances. Each AMP Packager Pod is
-associated with an NFS Server as its PersistentVolume which is created as a
-standard persistent disk defined by Google Kubernetes Engine.
+This application uses 2 replicas of the AMP Packager with a load balancer
+service in front that routes requests to the instances. The number of replicas
+is configurable.  All AMP Packager Pods are associated with an NFS Server as its
```
Comment (twifkak) [inline k8s/amppackager/README.md:37]: s/its/their

File: k8s/amppackager/README.md [hunk 2]
```diff
@@ -243,6 +245,10 @@ kubectl get svc -l app.kubernetes.io/name=$APP_INSTANCE_NAME --namespace "$NAMES
 This command shows the internal and external IP address of your AMP Packager
 service.
 
+You can follow [AMP Packager Productionizing][Production] for further details.
```
Comment (twifkak) [inline k8s/amppackager/README.md:248]: s/details/steps/ maybe?

File: k8s/amppackager/README.md (hunk 1 above)
Comment (banaag) [reply k8s/amppackager/README.md:37]: Done

File: k8s/amppackager/README.md (hunk 2 above)
Comment (banaag) [reply k8s/amppackager/README.md:248]: Done

**Pull request merged**